tests/unit/pubsub.tcl

```diff
@@ -504,3 +504,27 @@ start_server {tags {"pubsub network"}} {
     } {} {resp3}
 
 }
+
+start_server {tags {"pubsub external:skip logreqres:skip"}} {
+    # This test is for PUBLISH/PSUBSCRIBE.
+    # PUBLISH/SUBSCRIBE test is in obuf-limits named "Client output buffer hard limit is enforced".
+    test "PUBLISH/PSUBSCRIBE when client-output-buffer-limit hard limit is reached" {
```

Comment:
The comment "This test is for PUBLISH/PSUBSCRIBE" is not needed. It is very clear from the test case title. :-)

It weird that the same test for SUBSCRIBE is in another test suite. I think they should be together.

Why do we need to test it for PSUBSCRIBE and SUBSCRIBE separately? I think it is the same code path? In both cases, the client is evicted in the same way, by the addReply functions?

Reply:
Yes, actually it's the same code path. I add this test mainly because the test about SUBSCRIBE is in obuf-limits.tcl which seems to want to test what will happen if client output buffer hard limit is reached. If we want, we can also use other commands to achieve it, i.e get a very big key. The test I add wants to test what will happen if client output buffer hard limit is reached during PUBLISH, so I put it in pubsub.tcl. If you think it's redundant, I can remove it. : )

Reply:
I think we don't need the new tesy case.